### DISCUSSION TITLE
stop connection lines from merging

### DISCUSSION THREAD

**Discussion by simpsoneric** [terrastruct/d2#2122] [Q&A]
Using the tala layout engine, I have a diagram where I need to keep the connections independent.

https://play.d2lang.com/?script=SsvP50pKLOLiyk3MzOPiSsvPV9DVVQBxrBTyEnNTU0CyCKHi_NzUkozMvHQuQAAAAP__&layout=tala&

<img width="421" alt="image" src="https://github.com/user-attachments/assets/1bce325f-d2ac-4495-ad3d-e1250dd2b160">

This is a simple example above, but I have a shared resource with many different physical connections.  Is there a way to force tala to keep connections from merging?

This is somewhat related to the ideas in:

- https://github.com/terrastruct/d2/discussions/605

**Comment by alixander** [comment]
Hey @simpsoneric can you move this over to https://github.com/terrastruct/TALA ? Happy to answer this there
